### ISSUE TITLE
Shutdown is getting called twice

### ISSUE THREAD

**Issue by pedramphp** [krakenjs/kraken-js#402]
Looking at the code below, the shutdown keeps getting called if both of the below events gets fired.
```
 process.once('SIGTERM', close);
 process.once('SIGINT', close);
```
We are getting an error when we try to redeploy our node project.

https://github.com/krakenjs/kraken-js/blob/v1.0.x/middleware/shutdown.js#L30
```
==== ERROR ====
message: Not running
stack: Error: Not running
   at Server.close (net.js:1237:11)
   at EventEmitter.onshutdown
(/ebay/cronus/software/service_nodes/.ENVht7r82f541ib0.r1coltv-app__ENVht7r82f541ib0.r1coltv-app__ENVht7r82f541ib0-PHX-CLht7r93m039dpi-10.86.31.140/installed-packages/r1coltv/1.1.0_20150723165724182.unx/cronus/scripts/node_modules/kraken-js/lib/events.js:38:16)
   at EventEmitter.emit (events.js:98:17)
   at process.close
(/ebay/cronus/software/service_nodes/.ENVht7r82f541ib0.r1coltv-app__ENVht7r82f541ib0.r1coltv-app__ENVht7r82f541ib0-PHX-CLht7r93m039dpi-10.86.31.140/installed-packages/r1coltv/1.1.0_20150723165724182.unx/cronus/scripts/node_modules/kraken-js/middleware/shutdown.js:31:13)
   at process.g (events.js:180:16)
   at process.emit (events.js:117:20)
   at Signal.wrap.onsignal (node.js:790:46)
```

**Comment by jasisk** [comment]
Is this because both `SIGINT` and `SIGTERM` are firing, or are you seeing it because `SIGINT` is being captured twice (two `⌃C`s)?

Either way, you're absolutely right that we shouldn't be calling `server.close` twice.

**Comment by jasisk** [comment]
Closed by #417 #419
